docs/introduction/using-form-types.md

```diff
@@ -0,0 +1,255 @@
+If there is no customer set, `unregistered customer` text will be displayed instead.
+
+### [DisplayOnlyType](../../packages/framework/src/Form/DisplayOnlyType.php)
+Sometimes form needs to only display information but does not need to change and persist this data, for this usages
+there is `DisplayOnlyType` which does not map property onto `entity` and let you to display your own data.
+
+### [DisplayOnlyUrlType](../../packages/framework/src/Form/DisplayOnlyUrlType.php)
+For displaying custom URL based on routing system, there can be used `DisplayOnlyUrlType`.
+
+### [LocalizedFullWidthType](../../packages/framework/src/Form/LocalizedFullWidthType.php)
+For displaying localized field in vertical order of full-width label and inputs, there can be used `LocalizedFullWidthType`.
+
+### [OrderItemsType](../../packages/framework/src/Form/OrderItemsType.php)
+Displays editable table of OrderItems from provided Order.
+
+### [WarningMessageType](../../packages/framework/src/Form/WarningMessageType.php)
+Sometimes the form needs to contain some information that is important for viewer, for this usage there is  `WarningMessageType`
+that shows highlighted message with warning icon.
+
+### [LocalizedType](../../packages/framework/src/Form/Locale/LocalizedType.php)
+Compound type that renders one form of given type for each locale.
+Returns array indexed by locale.
+
+#### entry_type
+Defaults to `TextType::class`.
+It’s used to define what FormType should be used for the inner forms.
+
+#### entry_options
+An array of options that is used for every inner form.
+
+#### main_constraints
+An array of Constraints that is user for field with same locale as administration.
+
+### [CategoriesType](../../packages/framework/src/Form/CategoriesType.php)
+Displays a tree of all categories for given `domain_id` with checkboxes for each category created by `CategoryCheckBoxType` and returns array indexed by checked category ids.
+
+#### domain_id
+Required option that defines for what domain should the categories be listed.
+
+### [CategoryCheckboxType](../../packages/framework/src/Form/CategoryCheckboxType.php)
+Creates checkbox with and label with category name if name of the form is same as category id.
+Adds `visible`, `category_name`, `has_children` and `level` vars from category on domain for given `domain_id` to FormView so you can easily work with the checkbox.
+
+#### domain_id
+Required option that defines from what domain should the category be listed.
+
+### [ColorPickerType](../../packages/framework/src/Form/ColorPickerType.php)
+Displays text field with box of given color that shows color picker when clicked.
+
+### [DatePickerType](../../packages/framework/src/Form/DatePickerType.php)
+Displays field that shows date picker when clicked.
+
+#### format
+Defines in what format should be the date shown.
+DatePickerType has 2 constants that can be used:
+- FORMAT_PHP = 'dd.MM.yyyy'
+- FORMAT_JS = 'dd.mm.yy'
+Defaults to DatePickerType::FORMAT_PHP.
```

Comment:
maybe defaults to could be below title since it is like that for `Default options`